core/templates/pages/admin-page/services/admin-misc-tab-backend-api.service.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
The name of this service is not correct, I would suggest moving all the function to the existing `admin-backend-api.service` file. (OR find a better name of this service.)

Reply:
I cloned them into admin-backend-api.service